js/igv-create.js

```diff
@@ -497,6 +501,80 @@ var igv = (function (igv) {
 
     }
 
+    igv.displayChromosomePanel = function () {
+     var canvas = document.createElement('canvas');
+     canvas.id     = "scaleDisplay";
+     canvas.className = 'igv-chromosome-display-panel';
+     canvas.style.width  = document.body.clientWidth;
+     canvas.style.height = document.body.clientHeight/3;
+     canvas.style.position = "relative";
+     canvas.style.display = 'block';
+     canvas.style.overflow = 'auto';
+     canvas.style.float = 'left';
+     document.getElementById('igvControlDiv').appendChild(canvas);
+     igv.resizeDisplay();
+   }
+
+   igv.resizeDisplay = function () {
+     var canvas=document.getElementById('scaleDisplay');
+     canvas.width  = document.body.clientWidth;
+     canvas.height = document.body.clientHeight/3;
+     if (canvas.getContext) {
+       var ctx=canvas.getContext("2d");
+       var cntx=canvas.width*0.05;  //Will use to center the Chromosome Panel
+       igv.graphics.strokeLine(ctx,cntx,40,canvas.width-cntx,40);
+       igv.graphics.strokeLine(ctx,cntx,60,canvas.width-cntx,60);
+     var chromosomeArray = [249250621,243199373,198022430,191154276,180915260,171115067,159138663,146364022,141213431,135534747,135006516,133851895,115169878,107349540,102531392,90354753,81195210,78077248,59128983,63025520,48129895,51304566,155270560,59373566];
+     var totalBP=3095677412; // Sum of elements of chromosomeArray . In-bounds checked
```

Comment:
Do not calculate the sum manually. Use the function below instead
var totalBP = _.reduce(chromosomeArray, function(memo, num){ return memo + num; }, 0);